bot/internal/bot/changelog.go

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bot
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"regexp"
+	"strings"
+	"unicode"
+
+	"github.com/gravitational/trace"
+	"k8s.io/utils/strings/slices"
+)
+
+const NoChangelogLabel string = "no-changelog"
+const ChangelogPrefix string = "changelog: "
+const ChangelogTag string = "changelog"
+
+func (b *Bot) Changelog(ctx context.Context) error {
```

Comment:
Nit: `Changelog` is a very vague name for this method. Maybe `CheckChangelog` or something? Also, add godoc.